supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/RunStatus.scala

```diff
@@ -51,12 +51,16 @@ object RunStatus {
               zone: Option[String],
               instanceName: Option[String]): UnsuccessfulRunStatus = {
       val jesCode: Option[Int] = errorMessage flatMap { em => Try(em.substring(0, em.indexOf(':')).toInt).toOption }
-      if (errorCode == Status.ABORTED && jesCode.contains(JesAsyncBackendJobExecutionActor.JesPreemption)) {
-        Preempted(errorCode, jesCode, errorMessage, eventList, machineType, zone, instanceName)
-      } else if (errorCode == Status.CANCELLED) {
-        Cancelled(errorCode, jesCode, errorMessage, eventList, machineType, zone, instanceName)
-      } else {
-        Failed(errorCode, jesCode, errorMessage, eventList, machineType, zone, instanceName)
+
+      errorCode match {
+        case Status.ABORTED if jesCode.contains(JesAsyncBackendJobExecutionActor.JesPreemption) =>
+          Preempted(errorCode, jesCode, errorMessage, eventList, machineType, zone, instanceName)
+        case Status.UNKNOWN if errorMessage.exists(_.contains("failed to start due to preemption")) =>
```

Comment:
why does the one above get a constant and not this one?

Reply:
It's because the one above is a JES error code (ie an `Integer`) whereas this one is just a portion of an error message `String`. I can make it a constant if you prefer?